### PR TITLE
Revise some narrative and make the homepage shorter

### PR DESCRIPTION
**Pull request by szihai**
fix Issue #32.

### CODE REVIEW

File: themes/clean-landing/layouts/partials/implementations.html [hunk 1]
```diff
@@ -3,85 +3,27 @@
   <div class="content-wrapper">
     
     <div class="subtitle has-text-centered">
-      <h4>Implementations</h4>
+      <h4>Eco System</h4>
```
Comment (resouer) [inline themes/clean-landing/layouts/partials/implementations.html:6]: Ecosystem

File: config.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ params:
     url: /#overview
   - title: Examples
     url: /#examples
-  - title: Implementations
+  - title: Eco System
```
Comment (resouer) [inline config.yml:44]: Ecosystem

File: themes/clean-landing/layouts/partials/implementations.html [hunk 3]
```diff
@@ -3,85 +3,27 @@
...
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
+      <div class="column is-one-fifths is-one-fifths-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
-      <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
```
Comment (resouer) [inline themes/clean-landing/layouts/partials/implementations.html:13]: Please keep the original narrative in this section.

Comment (resouer) [reply themes/clean-landing/layouts/partials/implementations.html:13]: And we need to change another narrative above as we discussed.

File: config.yml (hunk 2 above)
Comment (szihai) [reply config.yml]: changed.

File: themes/clean-landing/layouts/partials/implementations.html (hunk 3 above)
Comment (szihai) [reply themes/clean-landing/layouts/partials/implementations.html:13]: changed

File: themes/clean-landing/layouts/partials/implementations.html (hunk 1 above)
Comment (szihai) [reply themes/clean-landing/layouts/partials/implementations.html]: changed

File: themes/clean-landing/layouts/partials/implementations.html [hunk 4]
```diff
@@ -3,85 +3,27 @@
...
-        <small><a href="https://crossplane.io">crossplane.io</a></small>
+      <div class="column is-four-fifths is-four-fifths-mobile">
+        <p><b>Crossplane OAM Kubernetes Runtime</b>: Crossplane implements OAM to deliver modern applications with cloud resources in a standard approach.</p>
+        <small><a href="https://github.com/crossplane/oam-kubernetes-runtime">Crossplane OAM K8s runtime</a></small>
```
Comment (vturecek) [inline themes/clean-landing/layouts/partials/implementations.html:14]: We should make sure the Crossplane folks are good with this change. @prasek

File: themes/clean-landing/layouts/partials/implementations.html [hunk 5]
```diff
@@ -3,85 +3,27 @@
...
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
+      <div class="column is-one-fifths is-one-fifths-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
-      <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
-        <small><a href="https://crossplane.io">crossplane.io</a></small>
+      <div class="column is-four-fifths is-four-fifths-mobile">
+        <p><b>Crossplane</b>: Crossplane implements OAM to deliver modern applications with cloud resources in a standard approach.</p>
```
Comment (resouer) [inline themes/clean-landing/layouts/partials/implementations.html:13]: Just copy from original section:

Crossplane is the open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.

others LGTM, thanks!

**Pull request merged**